public/components/security/users/components/create-user.tsx

```diff
@@ -0,0 +1,268 @@
+    password: [
+      { fn: () => (password === '' ? 'The password is required' : '') },
+      {
+        fn: () =>
+          !password.match(/^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[^A-Za-z0-9]).{8,64}$/)
+            ? 'The password must contain a length between 8 and 64 characters, and must contain at least one upper and lower case letter, a number and a symbol.'
+            : '',
+      },
+    ],
+    confirmPassword: [
+      { fn: () => (confirmPassword === '' ? 'The confirm password is required' : '') },
+      { fn: () => (confirmPassword !== password ? `Passwords don't match.` : '') },
+    ],
+  };
+
+  const validateField = field => {
+    const error = validations[field].reduce((result, validation) => {
+      return !!result ? result : validation.fn();
+    }, '');
+    setFormErrors({ ...formErrors, [field]: error });
+    return !!error;
+  };
+
+  const isValidForm = () => {
+    let errors = false;
+    Object.keys(validations).forEach(field => {
+      errors = errors || validateField(field);
+    });
+    return !errors;
+  };
+
+  const editUser = async () => {
+    if (!isValidForm()) {
+      ErrorHandler.warning('Please resolve the incorrect fields.');
+      return;
+    }
+
+    setIsLoading(true);
+
+    const userData: TCreateUser = {
+      username: userName,
+      password: password,
+      allow_run_as: allowRunAs,
+    };
+
+    try {
+      const user = await UsersServices.CreateUser(userData);
+      await addRoles(user.id);
+
+      ErrorHandler.info('User was successfully created');
+      closeFlyout(false);
+    } catch (error) {
+      ErrorHandler.handle(error, 'There was an error');
+      setIsLoading(false);
+    }
+  };
+
+  const addRoles = async userId => {
+    const formattedRoles = selectedRoles.map(item => {
+      return item.id;
+    });
+    if (formattedRoles.length > 0) await UsersServices.AddUserRoles(userId, formattedRoles);
+  };
+
+  const onChangeRoles = selectedRoles => {
+    setSelectedRole(selectedRoles);
+  };
+
+  const onChangeUserName = e => {
+    setUserName(e.target.value);
+  };
+
+  const onChangePassword = e => {
+    setPassword(e.target.value);
+  };
+
+  const onChangeConfirmPassword = e => {
+    setConfirmPassword(e.target.value);
+  };
+
+  const onChangeAllowRunAs = e => {
+    setAllowRunAs(e.target.checked);
+  };
+
+  return (
+    <EuiFlyout onClose={() => closeFlyout()}>
+      <EuiFlyoutHeader hasBorder={false}>
+        <EuiTitle size="m">
+          <h2>Create new user</h2>
+        </EuiTitle>
+      </EuiFlyoutHeader>
+      <EuiFlyoutBody>
+        <EuiForm component="form" style={{ padding: 24 }}>
+          <EuiPanel>
+            <EuiTitle size="s">
+              <h2>User data</h2>
+            </EuiTitle>
+            <EuiSpacer />
+            <EuiFormRow
+              label="User name"
+              isInvalid={!!formErrors.userName}
+              error={formErrors.userName}
+              helpText="Introduce a the user name for the user."
```

Comment:
typo:
from:
`Introduce a the user name for the user.`

to: 
`Introduce the user name for the user.`